db/organizations/delta_airlines.eno

```diff
@@ -0,0 +1,9 @@
+name: Delta Air Lines
+website_url: http://delta.com/
+privacy_policy_url: https://www.delta.com/us/en/legal/privacy-and-security
+privacy_contact: privacy@delta.com
+country: US
+description: Delta Air Lines, Inc. is a major airline in the United States headquartered in Atlanta, Georgia, operating nine hubs, with Hartsfield–Jackson Atlanta International Airport.
```

Comment:
Since the current sentence is incomplete, what about this version?

```suggestion
description: Delta Air Lines, Inc. is a major airline in the United States headquartered in Atlanta, Georgia. It ranks first in revenue and brand value among the world's largest airlines, and second by number of passengers carried, passenger miles flown, and fleet size.
```